frontend/src/components/AboutDialog/AboutDialog.tsx

```diff
@@ -0,0 +1,44 @@
+import Dialog from 'material-ui/Dialog'
+import * as React from 'react'
+import { SecureLink } from '../../utils/link'
+
+type Props = {
+    isOpen: boolean
+    onCloseClick: (buttonClicked: boolean) => void
+}
+
+export let AboutDialog: React.StatelessComponent<Props> = ({
+    isOpen, onCloseClick
+}) => {
+    let beaconLink = SecureLink('https://bayesimpact.org', 'Beacon Labs')
+    let emailLink = SecureLink('mailto:encompass@bayesimpact.org?subject=About%20Encompass', 'here')
+    let githubLink = SecureLink('https://github.com/bayesimpact/encompass', 'Github')
+    let osrmLink = SecureLink('https://github.com/bayesimpact/encompass', 'OSRM')
+    let usCensusLink = SecureLink('https://github.com/bayesimpact/encompass', 'U.S. Census data')
+
+    return <Dialog
+        title='About'
+        autoScrollBodyContent={true}
+        open={isOpen}
+        onRequestClose={onCloseClick}>
+        <div>
+            <div style={{ height: 140, display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
```

Comment:
Any reason why these styles (and others defined inline in this file) can't go into a corresponding css file?

Reply:
agreed. should definitely go in css file to be consistent